tests/foreman/cli/test_import.py

```diff
@@ -29,6 +36,16 @@ def csv_to_dataset(csv):
     ]
 
 
+def build_csv_file(header, rows=[]):
```

Comment:
The `mock` library provides good mocking facilities. You can even mock methods such as `open`. For some insight on the topic, see [How to Mock Python's `open` function](http://www.ichimonji10.name/blog/6/).